nondex-instrumentation/src/main/java/edu/illinois/nondex/instr/Instrumenter.java

```diff
@@ -203,6 +206,13 @@ public byte[] apply() {
                         }
                     });
         }
+        // add SpliteratorShuffler.class
```

Comment:
Should this be done only in some cases, like the `if-then-else` above?

Reply:
I can confirm for this `key` case, we do not have to do this.

We probably will need this for the `EntrySpliterator`, but I will look into this to confirm further.